src/pages/Cart/Cart.tsx

```diff
@@ -0,0 +1,69 @@
+import { FC } from 'react';
+import { Link } from 'react-router-dom';
+import { useAppContext } from '../../context/AppContext';
+import { ProductInCart } from '../../types/ProductInCart';
+import { ButtonBack } from '../../components/ButtonBack';
+import { CartItem } from '../../components/CartItem';
+
+import './cart.scss';
+
+type Props = {
+  products: ProductInCart[];
+};
+
+export const Cart: FC<Props> = ({
+  products,
+}) => {
+  const { inCartCount } = useAppContext();
+
+  const totalPrice = () => {
+    return products.reduce(
+      (acc: number, el: ProductInCart) => acc + el.price * el.count, 0,
+    );
+  };
+
+  const totalSum = totalPrice();
```

Comment:
No need in extra variable
```suggestion

  const totalSum = products.reduce(
      (acc: number, el: ProductInCart) => acc + el.price * el.count, 0,
    );
```